python/kdsource/geom.py

```diff
@@ -355,6 +355,33 @@ def load(mtree):
         return Lethargy(*params)
 
 
+class Wavelength(Metric):
+    def __init__(self):
+        """
+        Wavelength metric for energy.
+        Wavelength is defined as:
+        lambda [AA] = 9.045 / sqrt(E [meV])
+        """
+        super().__init__([0], ["l"], ["[AA]"], "AA")
+
+    def transform(self, ekins):
+        """Transform energy to lethargy."""
```

Comment:
I think this comment should be energy to wavelength. 
```suggestion
        """Transform energy to wavelenght."""
```